core_components/bot_core/guild_component_management.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 // handleInitialComponentStatusOnGuildJoin ensures that components that are enabled by default
-// are written to the database and - if newly written enabled by default.
+// are written to the entities and - if newly written enabled by default.
```

Comment:
Are they written to the "_entities_" or to the "_entities_"? Does this make a difference? Refactor issue?